### PR TITLE
[ospp] Adds MeasureAggregateFunctionService.Support API

### PR DESCRIPTION
**Pull request by StLeoX**
### <Feature description>
- [x] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.

[link](https://github.com/apache/skywalking/issues/8720#issuecomment-2196389729)

### CODE REVIEW

File: api/proto/banyandb/database/v1/schema.proto [hunk 1]
```diff
@@ -95,6 +96,8 @@ message FieldSpec {
   EncodingMethod encoding_method = 3 [(validate.rules).enum.defined_only = true];
   // compression_method indicates how to compress data during writing
   CompressionMethod compression_method = 4 [(validate.rules).enum.defined_only = true];
+  // aggregate_function indicates how to aggregate data
+  model.v1.AggregationFunction aggregate_function = 5;
```
Comment (wu-sheng) [inline api/proto/banyandb/database/v1/schema.proto:100]: @hanahmily I think this is not the function I mentioned. This AggregationFunction is targeting for query aggregation, this API changes should be stream flowing process aggregation, right?

Comment (hanahmily) [reply api/proto/banyandb/database/v1/schema.proto:100]: Our plan is to use the same functions to handle these scenarios. These functions are pure computing logic without any context related to the processes they are in.

File: api/proto/banyandb/model/v1/common.proto [hunk 2]
```diff
@@ -77,3 +77,17 @@ enum AggregationFunction {
...
+  MEASURE_AGGREGATE_UNSPECIFIED = 0;
+  MEASURE_AGGREGATE_MIN = 1;
+  MEASURE_AGGREGATE_MAX = 2;
+  MEASURE_AGGREGATE_COUNT = 3;
+  MEASURE_AGGREGATE_SUM = 4;
+  MEASURE_AGGREGATE_AVG = 5;
+  MEASURE_AGGREGATE_PERCENT = 6;
+  MEASURE_AGGREGATE_RATE = 7;
+  MEASURE_AGGREGATE_HISTOGRAM = 8;
+  MEASURE_AGGREGATE_PERCENTILE2 = 9;
```
Comment (hanahmily) [inline api/proto/banyandb/model/v1/common.proto:91]: @wu-sheng Would you mind if we call it 'percentile2', which aligns with the name defined in OAP?

Comment (wu-sheng) [reply api/proto/banyandb/model/v1/common.proto:91]: I think it is OK to align. Do you have percentile? I think we need that too.

Comment (StLeoX) [reply api/proto/banyandb/model/v1/common.proto:91]: Ok, I'll add that.

File: docs/api-reference.md [hunk 3]
```diff
@@ -690,6 +695,28 @@ Trace is the top level message of a trace.
...
 
+
+<a name="banyandb-model-v1-MeasureAggregate"></a>
+
+### MeasureAggregate
+
+
+| Name | Number | Description |
+| ---- | ------ | ----------- |
+| MEASURE_AGGREGATE_UNSPECIFIED | 0 |  |
```
Comment (wu-sheng) [inline docs/api-reference.md:706]: I think comments are required to generate docs here, to describe meanings of each function.

Comment (StLeoX) [reply docs/api-reference.md:706]: Sure

**Review by hanahmily — APPROVED** [review]

**Pull request merged**